packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -94,18 +94,35 @@ document.addEventListener('dart-app-ready', function (e) {
    styleSheet.parentNode.removeChild(styleSheet);
 });
 
+// Create a TrustedTypes policy so we can attach Scripts...
+var _ttPolicy;
```

Comment:
Can we use `let`? Or are there browsers we target that don't support it?

Reply:
We can, however there's quite a mix of let and var in this file :P